test/mocks/pscVerification.mock.ts

```diff
@@ -8,83 +9,101 @@ export const PSC_VERIFICATION_ID = "662a0de6a2c6f9aead0f32ab";
 export const UVID = "123abc456edf";
 export const SELF_URI = `/transactions/${TRANSACTION_ID}/persons-with-significant-control-verification/${PSC_VERIFICATION_ID}`;
 
-export const INITIAL_DATA: PscVerification = {
-    company_number: COMPANY_NUMBER
+export const INITIAL_PSC_DATA: PscVerificationData = {
+    companyNumber: COMPANY_NUMBER
 };
 
-export const INDIVIDUAL_DATA: PscVerification = {
-    company_number: COMPANY_NUMBER,
-    psc_appointment_id: PSC_VERIFICATION_ID,
-    verification_details: {
-        name_mismatch_reason: NameMismatchReason.MAIDEN_NAME,
-        verification_statements: [VerificationStatement.INDIVIDUAL_VERIFIED]
+export const LINKS: Links = {
+    self: SELF_URI,
+    validationStatus: ""
+};
+
+export const INDIVIDUAL_DATA: PscVerificationData = {
+    companyNumber: COMPANY_NUMBER,
+    pscAppointmentId: PSC_VERIFICATION_ID,
+    verificationDetails: {
+        verificationStatements: [VerificationStatementEnum.INDIVIDUAL_VERIFIED]
     }
 };
 
-export const PATCHED_INDIVIDUAL_PSC_ID_DATA: PscVerification = {
-    company_number: COMPANY_NUMBER,
-    psc_appointment_id: PSC_VERIFICATION_ID
+export const PATCH_INDIVIDUAL_DATA: PscVerificationData = {
+    companyNumber: COMPANY_NUMBER,
+    pscAppointmentId: PSC_ID
 };
```

Comment:
Could replace this and remove the import of PSC_ID above; test results seem unaffected.

```suggestion
    pscAppointmentId: PSC_VERIFICATION_ID
```